src/components/Loading/src/Loading.vue

```diff
@@ -1,34 +1,27 @@
 <template>
 	<div :class="$s.Loading">
-		<svg
+		<m-icon
 			:class="[
 				$s.Spinner,
 				$s[`size_${size}`],
```

Comment:
Does the size prop work properly with the new MIcon component? Looks like it was just setting height/width before, but you may need to pass it in as size prop where 'large' = 'xxlarge' icon size

Reply:
setting the size via a CSS class bypasses the prop, so i think this is okay, in the future i'd like to make a breaking change where i make the Loading sizes match the Icon sizes but we don't need to do that right now

Reply:
deployed styleguide with changes, large loading size is 64px as expected: https://square.github.io/maker/styleguide/themable-loading/#/Loading